src/LOOPSUID

```diff
@@ -209,8 +209,8 @@ DONTEVAL@COMPILE DOCOPY
                              ((Dec DEC) 
                                   'D)
                              (ERROR month "IS NOT A MONTH")))
-          
-          (* ;; "An unique ID consists of a front followed by a count.  The front is set up any time one enters -- It is unique because it contains the machine ID, the date, time of day in seconds, and has waited a second.  It then creates all of the UIDS in order from there.")
+
+         (* ;; "An unique ID consists of a front followed by a count.  The front is set up any time one enters -- It is unique because it contains the machine ID, the date, time of day in seconds, and has waited a second.  It then creates all of the UIDS in order from there.")
```

Comment:
it waits a second because? i think the thinking about parallel processes was confused.